src/theme/components/GlowEffect.tsx

```diff
@@ -0,0 +1,49 @@
+import { useWeb3React } from '@web3-react/core'
+import { SupportedChainId } from 'constants/chains'
+import { PropsWithChildren } from 'react'
+import styled, { css, DefaultTheme } from 'styled-components/macro'
+
+const getShadowForChainId = (theme: DefaultTheme, chainId?: SupportedChainId) => {
+  switch (chainId) {
+    case SupportedChainId.ARBITRUM_ONE:
+    case SupportedChainId.ARBITRUM_RINKEBY:
+      return theme.networkArbitrumShadow
+    case SupportedChainId.OPTIMISM:
+    case SupportedChainId.OPTIMISM_GOERLI:
+      return theme.networkOptimismShadow
+    case SupportedChainId.POLYGON:
+    case SupportedChainId.POLYGON_MUMBAI:
+      return theme.networkPolygonShadow
+    case SupportedChainId.CELO:
+    case SupportedChainId.CELO_ALFAJORES:
+      return theme.networkCeloShadow
+    default:
+      return theme.networkDefaultShadow
+  }
+}
```

Comment:
Now that I've spent some time in other places of the codebase, it could make more sense to make a map of ids to shadows and do something like `map[shadow] || defaultShadow`, which I think is similar conceptually to what we're already doing inside chain utils